rector.php

```diff
@@ -57,7 +56,7 @@
         PHPUnitSetList::PHPUNIT_100,
     ]);
 
-    $rectorConfig->parallel(240, 8, 1);
```

Comment:
btw: I can't make sense of a jobsize of 1. this should be pretty inefficient. nice its gone with this PR :)